src/cli/push.rs

```diff
@@ -12,31 +11,15 @@ pub struct Args {
 }
 
 pub fn execute(args: Args) {
-    let repo = common::get_araki_git_repo().unwrap_or_else(|err| {
-        eprintln!("Couldn't recognize the araki repo: {err}");
+    common::git_push(
+        "origin",
+        &[
+            "refs/heads/main",
```

Comment:
This will push the latest change on the main branch as well as the given tag. I'm not fully sure if that is the right behaviour. Should follow up on this with the work in https://github.com/openteams-ai/araki/issues/11

Reply:
Let's deal with this in a separate issue 👍